src/main/java/com/spotinst/sdkjava/model/bl/gcp/ElastigroupCapacityConfigurationGcp.java

```diff
@@ -0,0 +1,120 @@
+package com.spotinst.sdkjava.model.bl.gcp;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ElastigroupCapacityConfigurationGcp {
```

Comment:
Remove this class.

Reply:
already removed, not sure why we are seeing here.